source/mainnet/smart-contracts/onboarding-guide-ethereum-developers/faq.rst

```diff
@@ -0,0 +1,792 @@
+
+    If you insert your account address correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDAz9WdEy
+
+    The above request will return a transaction hash which you can look up on the block explorer.
+    You can only request CCD a single time for each account address via this API endpoint.
+    If you already submitted a request before, it will be an old transaction hash that is returned.
+
+    .. image:: ./images/onboarding_ethereum_developers_3.png
+        :width: 100 %
+
+    **Option 3:**
+    Alternatively, you can use `Postman <https://www.postman.com/>`_ and insert the following data before clicking the `Send` button.
+
+    .. image:: ./images/onboarding_ethereum_developers_4.png
+        :width: 100 %
+
+    **Option 4:**
+    If you have the curl package and the ``concordium-client`` tool installed on your Unix-like operating systems, you can request CCD to any of your alias account addresses.
+    If you already sent a previous request to the wallet proxy, you can not request any more CCD to the same account address.
+    Look up one of your alias account addresses instead and use it for your request.
+    The CCD will be credited to your canonical account address.
+
+    .. code-block:: console
+
+        $concordium-client account show-alias <YourAccountAddress> --alias <number>
+
+    If you insert your account address and a number correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $concordium-client account show-alias 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JaqovViXS4j --alias 17
+
+    This generates the output:
+
+    .. code-block:: console
+
+        The requested alias for address 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JaqovViXS4j is 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    Copy your alias address to the below command.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/<YourAliasAccountAddress>
+
+    If you insert your alias account address correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    **Option 5:** If you need plenty of CCD for large-scale testing,
+    please contact Concordium’s technical support via support@concordium.software.
+
+.. dropdown::  Is there something similiar to gas and transaction fees?
+
+    Yes, gas is called NRG (pronounced energy) on the Concordium chain.
+    The block limit is 3 million NRG. Transaction fees are fixed in Euro and are much
+    cheaper than on Ethereum.
+
+.. dropdown::  What networks can be used for testing?
+
+    Concordium has a testnet and stagenet for testing.
```

Comment:
The stagenet is internal. But perhaps we should mention that it is possible to get access to the protonet if they contact us?

Reply:
Changed it to:
    
    Concordium has a testnet, a stagenet, and a protonet for testing.
    The testnet is available publicly. Please contact us via Concordium’s technical support
    support@concordium.software if you have a special
    request for accessing other networks.